docs/src/bootstrap.md

```diff
@@ -275,6 +275,38 @@ spec:
     as queries are run as a superuser and can disrupt the entire cluster.
     An error in any of those queries interrupts the bootstrap phase, leaving the cluster incomplete.
 
+Moreover you can specify a list of secrets and/or configmaps which contains sql files that will be executed after the database is created and configured. These sql files will be executed as the *superuser* (`postgres`), connected to the database specified in the initdb:
+
+```yaml
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cluster-example-initdb
+spec:
+  instances: 3
+
+  bootstrap:
+    initdb:
+      database: app
+      owner: app
+      postInitApplicationSQLRefs:
+        configMapRefs:
+        - name: my-configmap
+          key: configmap.sql
+        secretRefs:
+        - name: my-secret
+          key: secret.sql
+  storage:
+    size: 1Gi
+```
+
+!!! Note
```

Comment:
I think we should say that we execute every SQL statement in a single exec on the server, and they will be executed according to the PostgreSQL semantics (described here https://www.postgresql.org/docs/current/protocol-flow.html#PROTOCOL-FLOW-MULTI-STATEMENT).

This means, i.e., that the user can add comments to the script but cannot use any `psql` internal command.